habitat_baselines/common/env_utils.py

```diff
@@ -0,0 +1,141 @@
+import random
+
+import numpy as np
+
+import habitat
+from habitat import Config, SimulatorActions, make_dataset
+
+
+class NavRLEnv(habitat.RLEnv):
+    def __init__(self, config_env, config_baseline, dataset):
+        self._config_env = config_env.TASK
+        self._config_baseline = config_baseline
+        self._previous_target_distance = None
+        self._previous_action = None
+        self._episode_distance_covered = None
+        super().__init__(config_env, dataset)
+
+    def reset(self):
+        self._previous_action = None
```

Comment:
Does `self._episode_distance_covered` also need to be set to `None`. It probably won't affect anything but just in case.

Reply:
I think setting to None makes sense. @abhiskk @mathfac did you guys choose to not reset it on purpose?